record/program/src/lib.rs

```diff
@@ -0,0 +1,13 @@
+//! Record program
+#![deny(missing_docs)]
+
+mod entrypoint;
+pub mod error;
+pub mod instruction;
+pub mod processor;
+pub mod state;
+
+// Export current SDK types for downstream users building with a different SDK version
+pub use solana_program;
+
+solana_program::declare_id!("CRUD3SXMKeggKN2FZePp8WLK2RpQvhp1LANGf6a5MLdu");
```

Comment:
Update grinded id to match new name?

Reply:
"record" is a lot of letters, still working on it :sweat_smile:

Reply:
Maybe next decade.  `Rec`? :)

Reply:
too easy! `rEcnBBXv6GqDJERL4yLVE5G2mcx7nVTkLHYeqw6mgpk` generated instantly

Reply:
Hold out for `Rec`, then ;)

Reply:
`ReciQBw6sQKH9TVVJQDnbnJ5W7FP539tPHjZhRF4E9r` :smile: